views/components/ConfigurationsList/index.jsx

```diff
@@ -427,6 +426,12 @@ const ConfigurationsList = ({ user, classes, theme }) => {
     }
   }
 
+  const onEditConfig = (configId) =>
+    navigate(routes.editConfiguration(configId))
+
+  const onViewConfig = (configId) =>
+    navigate(routes.viewConfiguration(configId))
```

Comment:
This is the second half to my other comment re: calling the prop with the config object itself. It is a more flexible approach that just calling it with the id.

```suggestion
  const onEditConfig = (config) =>
    navigate(routes.editConfiguration(config._id))

  const onViewConfig = (config) =>
    navigate(routes.viewConfiguration(config._id))
```

Reply:
I'm still not convinced since we haven't gotten to the EditConfigPage. I like that we have that option to go either way

Reply:
> I'm still not convinced since we haven't gotten to the EditConfigPage. I like that we have that option to go either way

can you explain this more? What are you not convinced about? What goes go either way mean?

Reply:
We can change this in the future if we have the need to send the entire object, right now this is just routing to a different page and that page uses the id

Reply:
ok I think this is just a fundamental disagreement. We're changing this implementation and, IMO, it's nice to add flexibility while doing that in a way that requires 0 additional effort. By doing that our future code change is smaller. I won't belabor the point more. If you prefer this style you can make the decision you want to make here.